nav2_system_tests/src/system/test_system_launch.py

```diff
@@ -33,20 +35,25 @@
 from launch_testing.legacy import LaunchTestService
 
 from nav2_common.launch import RewrittenYaml
+from nav2_simple_commander.utils import kill_os_processes
 
 
 def generate_launch_description():
-    map_yaml_file = os.getenv('TEST_MAP')
-    world = os.getenv('TEST_WORLD')
+    sim_dir = get_package_share_directory('nav2_minimal_tb3_sim')
+    nav2_bringup_dir = get_package_share_directory('nav2_bringup')
+    ros_gz_sim_dir = get_package_share_directory('ros_gz_sim')
+
+    world_sdf_xacro = os.path.join(sim_dir, 'worlds', 'tb3_sandbox.sdf.xacro')
+    robot_sdf = os.path.join(sim_dir, 'urdf', 'gz_waffle.sdf')
```

Comment:
I think this is `.xacro` now if you rebase into main from the changes in https://github.com/ros-navigation/nav2_minimal_turtlebot_simulation/pull/11 FYI (probably other launch files too). Just be aware that this change is coming :-)

Reply:
this will be a real localization test though. What if you add both there after I correct the test I have ported with the new xacro, then I will delete it from there.

Reply:
I think you can leave it as is, I'm just giving you an FYI that this will change once I get a chance on Friday to test the multirobot fixes to merge those in. So at that point, these would need to be updated once merged.

If this is merged before I make the changes, then I'll fix these tests. If its not merged, then you'll need to fix them. Its a race! :laughing: